src/folderitemdelegate.cpp

```diff
@@ -372,7 +372,8 @@ QWidget* FolderItemDelegate::createEditor(QWidget* parent, const QStyleOptionVie
         // ensure that its background isn't transparent (on the side-pane)
         QWidget* editor = QStyledItemDelegate::createEditor(parent, option, index);
         QPalette p = editor->palette();
-        p.setColor(QPalette::Base, QApplication::palette().color(QPalette::Base));
+        p.setColor(QPalette::Text, qApp->palette().text().color());
```

Comment:
@PCMan It is needed because we set the text color of the Places view to the window text color and it'll be inherited by the editor, while we need the base text color with the editor. Otherwise, the editor text might be invisible with unusual themes like that of the report in question (although such themes won't be practical at all).